components/homme/cmake/machineFiles/cori-knl.cmake

```diff
@@ -13,31 +13,7 @@ SET (HDF5_DIR $ENV{HDF5_DIR} CACHE FILEPATH "")
 
```

Comment:
by some reason, i cannot build on cori with pnetcdf, and have to have 
SET (WITH_PNETCDF FALSE CACHE FILEPATH "") . i don't see other significant differences between this cmake file and what i use.

Reply:
which pnetcdf module are you using?  It works for me with cray-parallel-netcdf/1.6.1